samples/timelapse-sample/timelapse_summary.h

```diff
@@ -0,0 +1,13 @@
+
+#include <PipeLine.h>
+
+class TimelapsePipeline {
+public:
+  TimelapsePipeline();
+  bool setupPipeline();
+  bool startPipeline();
+  bool stopPipeline();
+
+private:
+  PipeLine pipeline;
```

Comment:
add a method called test, and call from test added in timelapse-sample/ - it should use step - not the pipeline

Reply:
Done.